File: Jaar_1/JavaEssentials/H1/src/be/pxl/h1/opdracht9/Thermometer.java
package be.pxl.h1.opdracht9;

public class Thermometer {
    private double temperature;

    public double getFahrenheit() {
        return (9 / 5.0) * temperature + 32;
    }

    public double getTemperature() {
        return temperature;
    }

    public void setTemperature(double temperature) {
        this.temperature = temperature;
    }
}

File: Jaar_1/NETEssentials/oefeningen/h10/oef_10.2_Thermometer/Thermometer.cs
namespace oef_10._2_Thermometer
{
    public class Thermometer
    {
        private int _temperature;
        private int _max;
        private int _min;

        public void SetTemperature(int temperature)
        {
            _temperature = temperature;
            if (_max < _temperature)
            {
                _max = temperature;
            }
            if (_min > _temperature)
            {
                _min = temperature;
            }
        }

        public void Reset()
        {
            _temperature = 0;
            _max = 0;
            _min = 0;
        }

        public int GetMax()
        {
            return _max;
        }

        public int GetMin()
        {
            return _min;
        }
    }
}
